flags.json

```diff
@@ -1,9 +1,9 @@
 {
   "documentSymbol": {
-    "percent": 0
+    "percent": 0.01
   },
 
   "semanticHighlighting": {
-    "percent": 0
+    "percent": 0.01
```

Comment:
Can I just double check. The percentage is expressed as a decimal rather than an integer?

Reply:
Yep, it's basically `parseInt("2 random bytes", 16) / 65536` (in 0-1 range) and that's checked against these flag values:

https://github.com/NomicFoundation/hardhat-vscode/blob/fe6254ad467d0eb193f512f97008c387b63e9988/server/src/services/initialization/featureFlags.ts#L71-L73